.adminbro/.entry.js

```diff
@@ -1 +1,3 @@
-AdminBro.UserComponents = {};
```

Comment:
I still think this file is not necessary, and its output is a bit dirty. The adminbro's dev recommended to leave it out of the repo.